stratum/hal/lib/barefoot/bf_pal_wrapper.cc

```diff
@@ -276,6 +289,20 @@ bool BFPalWrapper::PortIsValid(int unit, uint32 port_id) {
   return (bf_pal_port_is_valid(unit, port_id) == BF_SUCCESS);
 }
 
+::util::Status BFPalWrapper::PortLoopbackModeSet(int unit, uint32 port_id,
+                                                 LoopbackState loopback_mode) {
+  ASSIGN_OR_RETURN(bf_loopback_mode_e lp_mode, LoopbackModeToBf(loopback_mode));
```

Comment:
Could we ignore the loopback mode UNKNOWN here? Then the other checks are not needed.

Reply:
It should be fine to return OK here if it is unknown, will do.